modules/auxiliary/admin/http/netgear_PNPX_GetShareFolderList_auth_bypass.rb

```diff
@@ -0,0 +1,127 @@
+##
```

Comment:
Module names are consistently snake_case where these should be all lowercase.

Reply:
Fixed in https://github.com/gwillcox-r7/metasploit-framework/commit/810fa6be6e3113d440843cbde5d00cb1b131e362